templates/spring-kotlin/files/src/main/kotlin/sh/platform/template/spring/kotlin/Application.kt

```diff
@@ -0,0 +1,12 @@
+package sh.platform.template.spring.kotlin
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
+
+@SpringBootApplication(scanBasePackages = arrayOf("sh.platform.template.spring.kotlin"),
+        exclude = arrayOf(SecurityAutoConfiguration::class))
+class Application
+fun main(args: Array<String>) {
```

Comment:
To my eye this looks like a missing { after Application, but that could just be me not knowing how Kotlin does things.  Verify?

Reply:
Yeap, it verifies.
It works.

Reply:
Kotlin is weird...